Makefile

```diff
@@ -3,167 +3,105 @@
-	@xcodebuild -scheme Pillarbox-demo -project ./Demo/Pillarbox-demo.xcodeproj -destination generic/platform=iOS -derivedDataPath ./.build/derived-data clean build &> /dev/null
-	@pkgx periphery scan --project ./Demo/Pillarbox-demo.xcodeproj --schemes Pillarbox-demo --targets Pillarbox-demo --skip-build --index-store-path ./.build/derived-data/Index.noindex/DataStore/
-	@echo "... done.\n"
+	@Scripts/find-dead-code.sh
 
 .PHONY: help
 help:
-	@echo "The following targets are available:"
-	@echo ""
-	@echo "   all                                Default target"
-	@echo ""
-	@echo "   fastlane                           Run fastlane"
-	@echo ""
-	@echo "   archive-demo-ios                   Archive the iOS demo (for all configurations)"
-	@echo "   archive-demo-tvos                  Archive the tvOS demo (for all configurations)"
-	@echo ""
-	@echo "   deliver-demo-nightly-ios           Deliver a demo nightly build for iOS"
-	@echo "   deliver-demo-nightly-tvos          Deliver a demo nightly build for tvOS"
-	@echo ""
-	@echo "   deliver-demo-release-ios           Deliver a demo release build for iOS"
-	@echo "   deliver-demo-release-tvos          Deliver a demo release build for tvOS"
-	@echo ""
-	@echo "   test-streams-start                 Start servicing test streams"
-	@echo "   test-streams-stop                  Stop servicing test streams"
-	@echo ""
-	@echo "   test-ios                           Build and run unit tests for iOS"
-	@echo "   test-tvos                          Build and run unit tests for tvOS"
-	@echo ""
-	@echo "   check-quality                      Run quality checks"
-	@echo "   fix-quality                        Fix quality automatically (if possible)"
-	@echo ""
-	@echo "   git-hook-install                   Use hooks located in ./hooks"
-	@echo "   git-hook-uninstall                 Use default hooks located in .git/hooks"
-	@echo ""
-	@echo "   spm-reload                         Reload SPM dependencies"
-	@echo "   clean-imports                      Remove useless imports from the project"
-	@echo "   find-dead-code                     Find dead code"
-	@echo ""
-	@echo "   help                               Display this help message"
+	@echo "Available targets:"
+	@echo
+	@echo "Default:"
+	@echo "  all                            Default target"
+	@echo
+	@echo "Archive:"
+	@echo "  archive-demo-ios               Archive iOS demo [MODE=\"--non-interactive\"]"
+	@echo "  archive-demo-tvos              Archive tvOS demo [MODE=\"--non-interactive\"]"
+	@echo
+	@echo "Deliver:"
+	@echo "  deliver-demo-nightly-ios       Deliver nightly iOS demo build [MODE=\"--non-interactive\"]"
+	@echo "  deliver-demo-nightly-tvos      Deliver nightly tvOS demo build [MODE=\"--non-interactive\"]"
+	@echo "  deliver-demo-release-ios       Deliver release iOS demo build [MODE=\"--non-interactive\"]"
+	@echo "  deliver-demo-release-tvos      Deliver release tvOS demo build [MODE=\"--non-interactive\"]"
```

Comment:
Non-interactive mode is mostly for internal use. Not sure it should be visible in documentation.